extractor/filesystem/os/dpkg/dpkg_test.go

```diff
@@ -886,6 +1309,51 @@ func TestExtractNonexistentOSRelease(t *testing.T) {
 	}
 }
 
+func TestExtractNonexistentOSReleaseOpkg(t *testing.T) {
```

Comment:
We didn't introduce any new logic for OPKG-specific OS release parsing so I think we can skip this test